### PR TITLE
Merge develop into feature/coupled-crow 

### PR DESCRIPTION
**Pull request by JessicaMeixner-NOAA**
This merges the develop branch into feature/coupled-crow.  Beyond running what was already running in feature/coupled-crow it did not do any hand-merging of updates in the forecast script or adding other jobs to the crow workflow etc. 

This was tested on orion: 
/work/noaa/marine/jmeixner/p7mergedev/test01/COMROOT/test01/gfs.20130401 
to a baseline of the current (minus the update from today) feature/coupled-crow: 
/work/noaa/marine/jmeixner/p7baseline/baseline01/COMROOT/test01/gfs.20130401

All diffs were given the okay by @jiandewang

### CODE REVIEW

File: sorc/machine-setup.sh [hunk 1]
```diff
@@ -56,16 +56,16 @@ elif [[ $hname = 'hpc.msstate.edu' ]] ; then
 elif [[ -d /scratch1 ]] ; then
     # We are on NOAA Hera
     if ( ! eval module help > /dev/null 2>&1 ) ; then
-	echo load the module command 1>&2
+        echo load the module command 1>&2
         source /apps/lmod/lmod/init/$__ms_shell
     fi
-    target=hera
+    target=orion
```
Comment (WalterKolczynski-NOAA) [inline sorc/machine-setup.sh]: This section should still be for Hera, no?

**Review by WalterKolczynski-NOAA — CHANGES_REQUESTED** [review]

File: sorc/machine-setup.sh (hunk 1 above)
Comment (WalterKolczynski-NOAA) [reply sorc/machine-setup.sh]: (there are more changes to orion on subsequent lines that will need to be changed back)

Comment (JessicaMeixner-NOAA) [reply sorc/machine-setup.sh]: Thanks @WalterKolczynski-NOAA I'll take a look when I get a chance.  I'm guessing this also means this PR should be tested on all the machines before merging as well to make sure other platform dependent bugs don't creep in unnoticed as it's tons of changes.

Comment (JessicaMeixner-NOAA) [reply sorc/machine-setup.sh]: @WalterKolczynski-NOAA I reverted back to what was in feature/coupled-crow. 

Any other red-flags for this branch or are we just at the stage where this branch needs to be run on all platforms?

Comment (WalterKolczynski-NOAA) [reply sorc/machine-setup.sh]: I think just testing. Looked pretty good overall.

**Comment by JessicaMeixner-NOAA** [comment]
Hera test completed: /scratch2/NCEPDEV/climate/Jessica.Meixner/p7mergedev/test01/COMROOT/test01/gfs.20130401

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Pull request merged**